### PR TITLE
7240 - Address compiler warning messages

### PR DESCRIPTION
**Pull request by kellykelly3**

### CODE REVIEW

File: KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/Bundle.properties-MERGED [hunk 1]
```diff
@@ -38,7 +37,7 @@ KeywordSearchResultFactory.createNodeForKey.noResultsFound.text=No results found
 KeywordSearchResultFactory.query.exception.msg=Could not perform the query 
 OpenIDE-Module-Display-Category=Ingest Module
 
-OpenIDE-Module-Long-Description=Keyword Search ingest module.\n\nThe module indexes files found in the disk image at ingest time.\nIt then periodically runs the search on the indexed files using one or more keyword lists (containing pure words and/or regular expressions) and posts results.\n\nThe module also contains additional tools integrated in the main GUI, such as keyword list configuration, keyword search bar in the top-right corner, extracted text viewer and search results viewer showing highlighted keywords found.
+OpenIDE-Module-Long-Description=Keyword Search ingest module.\n\nThe module indexes files found in the disk image at ingest time.\nIt then periodically runs the search on the indexed files using one or more keyword lists (containing pure words and/or regular expressions) and posts results.\n\n\The module also contains additional tools integrated in the main GUI, such as keyword list configuration, keyword search bar in the top-right corner, extracted text viewer and search results viewer showing highlighted keywords found.
```
Comment (rcordovano) [inline KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/Bundle.properties-MERGED:40]: Is this making the "T" into a tab?

This file is really not part of this PR.

**Review by rcordovano — CHANGES_REQUESTED** [review]

File: thunderbirdparser/ivy.xml [hunk 2]
```diff
@@ -14,5 +14,6 @@
         <dependency conf="autopsy->default" org="com.googlecode.ez-vcard" name="ez-vcard" rev="0.10.5"/>
         <dependency conf="autopsy->default" org="com.github.mangstadt" name="vinnie" rev="2.0.2"/>
         <dependency org="com.google.guava" name="guava" rev="19.0"/>
+        <dependency org="commons-validator" name="commons-validator" rev="1.6"/>
```
Comment (rcordovano) [inline thunderbirdparser/ivy.xml:17]: This is not related to the topic of this PR.

**Review by rcordovano — CHANGES_REQUESTED** [review]

File: thunderbirdparser/ivy.xml (hunk 2 above)
Comment (kellykelly3) [reply thunderbirdparser/ivy.xml:17]: @rcordovano this was to fix a warning.

**Review by rcordovano — APPROVED** [review]
